.github/workflows/azure_deploy.yml

```diff
@@ -0,0 +1,155 @@
+name: Deploy to Zooniverse Static
+
+on:
+    # Run this workflow from other workflows
+    workflow_call:
+      inputs:
+        commit_id: 
+          description: 'HEAD commit hash'
+          required: true
+          type: string
+        deploy_url:
+          description: 'Deployed URL'
+          required: true
+          type: string
+        job_name:
+          description: 'Job name to report on'
+          required: true
+          type: string
+        script:
+          description: 'npm build script'
+          required: true
+          type: string
+        source:
+          default: './dist'
+          description: 'Source directory'
+          required: false
+          type: string
+        target:
+          description: 'Path to target folder'
+          required: true
+          type: string
+        title:
+          description: 'Notification title'
+          required: true
+          type: string
+      secrets:
+        creds:
+          required: true
+        slack_webhook_url:
+          required: true
+      outputs:
+        status:
+          description: 'Deploy status'
+          value: ${{ jobs.deploy.outputs.status }}
```

Comment:
I think this is left over from a time before I found out that `job_id.result` exists, to get the status of a completed job. It's not used anywhere.